dashboard/test/models/user_test.rb

```diff
@@ -741,6 +741,17 @@ class UserTest < ActiveSupport::TestCase
     assert_equal '21+', user.age
   end
 
+  test 'changing oauth user from student to teacher with different email is allowed' do
```

Comment:
This test is incorrect and will fail (we decided to require the opposite, and explicitly disallow this change in user type unless the email matches), I'm updating the test now.